filters.go

```diff
@@ -34,7 +34,9 @@ func (p *Pipe) MatchRegexp(re *regexp.Regexp) *Pipe {
 	})
 }
 
-// Replace reads from the pipe, and returns a new pipe containing replaces lines
+// Replace reads from the pipe, and returns a new pipe
```

Comment:
Could you wrap these comments to 80 columns, for consistency with other comments? I use the Rewrap plugin in VS Code for this, which saves a lot of manual reformatting.

Reply:
Yep, okay:)